test/dynamo/test_misc.py

```diff
@@ -828,6 +828,48 @@ def forward(self, arg0_1: "f32[3]", arg1_1: "f32[3]", arg2_1: "f32[3]", arg3_1:
             cleanup_op("mylib::foo")
             del lib
 
+    def test_auto_functionalize_tensorlist(self):
+        try:
+            lib = torch.library.Library("mylib", "FRAGMENT")
```

Comment:
use torch._library.scoped_library in tests

Reply:
cc @kit1980 does torchlint get triggered on PRs?

Reply:
TorchFix should trigger and fail this, let me doublecheck.

Reply:
@zou3519 all flake8 lints are ignored for the file because line 7285 has "flake8: noqa".
It doesn't work as somebody who added that thought it works, it just unconditionally excludes the whole file from the linter.

Reply:
thanks for pointing that out

Reply:
Created an issue https://github.com/pytorch/pytorch/issues/127352

Reply:
updated